### PR TITLE
[DONE] Dirk hookup photoview

### PR DESCRIPTION
**Pull request by d-weeteling**

### CODE REVIEW

File: src/components/views/MainView.jsx [hunk 1]
```diff
@@ -39,40 +42,48 @@ class MainViewComponent extends Component {
   }
 
   render() {
-    let component = null;
+    const { getPhotoForSelectedParcel } = this.props;
     switch (this.props.currentView) {
       case "MapSearchView":
-        component = (
+        return (
```
Comment (gijs) [inline src/components/views/MainView.jsx]: I prefer this 👍 👍

Comment (d-weeteling) [inline src/components/views/MainView.jsx]: Yep!

**Comment by d-weeteling** [comment]
[[photos] [front-end] Photo in zowel PhotoView en DetailView [1]](https://trello.com/c/NyhRC1mK/1700-photos-front-end-photo-in-zowel-photoview-en-detailview-1)

**Comment by d-weeteling** [comment]
[Show spinners where relevant [0.5]](https://trello.com/c/3vHEh3x3/1895-show-spinners-where-relevant-05)

**Pull request merged**